debian/noembed/nginx/sites/build.whatwg.org.conf

```diff
@@ -0,0 +1,16 @@
+server {
+  listen 443 ssl http2 default_server;
```

Comment:
```
listen 443 ssl http2 default_server;
```

Because we already have a server block in `noembed/nginx/sites/participate.whatwg.org.conf` that has `listen 443`, nginx won’t let us also have it it here. I’ll make some changes to deal with that, and push them to the branch.